doc/src/devdocs/eval.md

```diff
@@ -4,9 +4,9 @@ One of the hardest parts about learning how the Julia Language runs code is lear
 the pieces work together to execute a block of code.
 
 Each chunk of code typically makes a trip through many esoteric acronyms such as (in no particular
```

Comment:
Yeah, I noticed that when I was going through here. Perhaps I should change it to:
> Each chunk of code typically makes a trip through many steps with potentially unfamiliar names, such as (in no particular order) ...

Calling C++ "esoteric" is a little odd IMO. 😛